book/service_container.rst

```diff
@@ -150,10 +150,10 @@ be specified in YAML, XML or PHP:
     ``config_dev.yml`` for the ``dev`` environment or ``config_prod.yml``
     for ``prod``).
 
-An instance of the ``Acme\HelloBundle\Mailer`` object is now available via
-the service container. The container is available in any traditional Symfony
-controller where you can access the services of the container via the ``get()``
-shortcut method::
+An instance of the ``AppBundle\Mailer`` class is now available via the service
+container. The container is available in any traditional Symfony controller
```

Comment:
I know it's not related to this PR, but this phrase could be reworded:

```
The container is available in any traditional Symfony controller
where you can access the services of the container via the ``get()`` shortcut
method::
```

Probably it would be better to say that it's available in any controller extending from the base controller provided by Symfony.